notebooks/tutorials/landslides/Synthetic_Landslide_Component_Default.ipynb

```diff
@@ -0,0 +1,1024 @@
+{
```

Comment:
We've been saving notebooks without output to reduce the size of the repo (and better be able to track real changes in notebooks). Can you save all notebooks without output.

Reply:
Yes! This is done.